### ISSUE TITLE
[Bug]: Could not pass event EntityPickupItemEvent to JukeboxExtendedReborn v1.1.3 java.lang.IllegalArgumentException: Cannot drop air

### ISSUE THREAD

**Issue by WataLaker37821** [spartacus04/jext-reborn#108]
### Describe the bug

I saw sudden error in console, my helper says he just played Pigstep disc and then stopped it, i use GUI system for jukeboxes

### Steps to reproduce

Just an error in a console, not sure why

### Expected behaviour

no errors in console

### Logs

https://pastebin.com/Dqq4Tp7b

### Server Version

1.18.2

### Server Type

Purpur

### Discs.json

```json
[
    {
        "title": "Odyssey",
        "author": "RazMusic",
        // duration of the disc file in seconds
        "duration": 1000,
        // disc name in sounds.json, example: 
        "disc-namespace": "kevinmacleod.odyssey",
        // model data of the disc
        "model-data": 88,
        // true if allows creeper drops, false otherwise
        "creeper-drop": true,
        "lores": [
            "lore added by disc",
            "each entry is a new line"
        ],
        // loottables in which the disc can be found
        "loot-tables": [
            "chests/ancient_city"
        ],
        // loottables in which the disc fragments can be found
        "fragment-loot-tables": [
            "chests/ancient_city"
        ]
    },
    {
        "title": "Cipher",
        "author": "RazMusic",
        "duration": 1000,
        "disc-namespace": "kevinmacleod.cipher",
        "model-data": 89,
        "creeper-drop": true,
        "lores": [],
        "loot-tables": [],
        "fragment-loot-tables": []
    },
    {
        "title": "Infiniteamethyst",
        "author": "RazMusic",
        "duration": 1000,
        "disc-namespace": "lenaraine.infiniteamethyst",
        "model-data": 90,
        "creeper-drop": true,
        "lores": [],
        "loot-tables": [],
        "fragment-loot-tables": []
    },
    {
        "title": "AlreadyDead",
        "author": "RazMusic",
        "duration": 1000,
        "disc-namespace": "razfill.alreadydead",
        "model-data": 91,
        "creeper-drop": true,
        "lores": [],
        "loot-tables": [],
        "fragment-loot-tables": []
    }
]
```

### Resource pack

i cant upload it cos i use Itemsadder and it is hosted by my server's additional port

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.
- [X] I agree to the fact that if this issue becomes inactive it will be closed.

**Comment by WataLaker37821** [comment]
full server log: https://pastebin.com/1g9YWUC1

**Comment by WataLaker37821** [comment]
i figured out that this error appears when jukebox breaks while disc Pigstep is playing

**Comment by spartacus04** [comment]
Hi, i just fixed this in [ee9f92f](https://github.com/spartacus04/jext-reborn/commit/ee9f92fadf1e82c261e7e1cabb7e0b45570421b4), it should be implemented in the next update
